mco_ovs_supervisor.yml

```diff
@@ -25,7 +25,7 @@ spec:
+            source: data:text/plain;charset=utf-8;base64,${base64_script_content}
```

Comment:
Rename this file to mco_ovs_supervisor.yaml.in to make it clear that this is **not** a file that users should apply. Then, in the example command to render it, only there create the mco_ovs_supervisor.yml file with the rendered content

Reply:
I can add this to the envsubst command output and create the new files, but we sure we want this? don't we want the same files? 

If so, Should we name the files in the repo mco_ovs_workers_template? and the output should be the current file name?

Reply:
We should make it hard for users to apply this YAML without reading the YAML and thus applying unrendered template.

Also, if you overwrite the file, then you cannot run the command again, you would have to revert changes in your source tree.

The output should be the current name. For the template, I'd prefer mco_ovs_workers.yml.in or mco_ovs_workers.yml.tmpl. But if you go with mco_ovs_workers_template, I'm ok with that